scene/gui/control.cpp

```diff
@@ -181,6 +183,47 @@ Size2 Control::_edit_get_minimum_size() const {
 }
 #endif
 
+// Editor integration.
```

Comment:
Maybe add a line after such comments to make it clear that they create a section. Otherwise to me a single line comment touching a method declaration only qualifies that single method.